src/librustc_metadata/diagnostics.rs

```diff
@@ -56,6 +56,19 @@ you want. Example:
 ```
 "##,
 
+E0463: r##"
+A plugin/crate was called but cannot be found. Erroneous code example:
+
+```
+#![feature(plugin)]
+#![plugin(cookie_monster)] // error: can't find crate for `cookie_monster`
+```
+
+In Rust, plugins are handled the same way as crate. You need to link your
```

Comment:
You need to link your code to the relevant crate in order to be able to use it (through Cargo or ....). Plugins are crates as well, and you link to them the same way.